conf/wagtaildocs-cloudflare-worker.js

```diff
@@ -13,11 +13,11 @@ versions over the current stable one.
 
 // The hostname we will fetch documentation pages from. This is configured in readthedocs
 // as an alternative (non-default) hostname with HTTPS disabled; this avoids any issues
-// with the docs.wagtail.io DNS and SSL certificate being handled by us vs. readthedocs.
+// with the docs.wagtail.org DNS and SSL certificate being handled by us vs. readthedocs.
 const originDomain = 'wagtaildocs.demozoo.org';
```

Comment:
Thought: Should we take this opportunity to use a wagtail domain here?

Reply:
Yep, I'd go along with that! `docs-internal.wagtail.org` or something.